src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Index.txt

```diff
@@ -0,0 +1,41 @@
+>> Index([10, 20, 30, 40, 50], 3)
+{Value:30}
+
+>> Index([1, 2, 3, 4, 5], 6)
+#Error
```

Comment:
`#Error(Kind=<the error kind returned by this function>)` - we can now validate the error kinds, and it is good to have it in the tests as the "specification". Same for the next test

Reply:
I specified the error kinds but it doesn't appear to make a difference. I can put anything as the kind and the test cases with an expected error will still pass.

Reply:
The test logic is ignoring the kinds, good catch. PR to fix this: https://github.com/microsoft/Power-Fx/pull/211